posthog/tasks/exporter.py

```diff
@@ -4,7 +4,7 @@
 from posthog.models import ExportedAsset
 
 
-@app.task(retries=3)
```

Comment:
I'm not confident this will fix the underlying issue as this task will be retried only if an exception is raised. 

_This is good but not enough_ to retry the task in the unfortunate event the system where Celery is running (local laptop, pod, dedicated server, etc..) vanishes without any notice. To handle also this specific use case we could probably use some of the ack functionalities from celery ([see docs](https://docs.celeryq.dev/en/latest/faq.html#faq-acks-late-vs-retry))